apps/address-service/domains/common/utils/services/suggest/providers/DadataSuggestionProvider.js

```diff
@@ -171,10 +171,23 @@ class DadataSuggestionProvider extends AbstractSuggestionProvider {
             },
         )
 
+        //TODO?(nas): add some calls counter (maybe datadog)
+
         const status = result.status
         if (status === 200) {
             const response = await result.json()
             return get(response, 'suggestions', [])
+        } else if (status === 403) {
+            /**
```

Comment:
Maybe return error here?

Reply:
This may be an error about the limit being exceeded. I want to create the rotation of API keys here.